ow_lander/scripts/arm_action_servers.py

```diff
@@ -175,6 +171,34 @@ def now_from_start(start):
             self._server.set_aborted(self._result)
 
 
+class StopActionServer(object):
+
+    def __init__(self, name):
+        self._action_name = name
+        self._server = actionlib.SimpleActionServer(self._action_name,
+                                                    ow_lander.msg.StopAction,
+                                                    execute_cb=self.on_stop_action,
+                                                    auto_start=False)
+        self._server.start()
```

Comment:
Please move these two lines to the end of `__init__` to avoid the bug that's addressed by #221